src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -235,7 +234,14 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     }
     else
     {
-      macroSubstitute(&payload, httpInfo.payload, ce);
+      bool b = macroSubstitute(&payload, httpInfo.payload, ce);
+
+      if (b == false)
```

Comment:
A more compact alternative (used in other places in our code sometimes):

```
      if (!macroSubstitute(&payload, httpInfo.payload, ce))
      {
        LM_E(("Bad Input (not sending NotifyContextRequest: payload too large)"));
        return paramsV;  // empty vector
      }
```

Just a suggestion. It is ok also the current way.